run_all_experiments.py

```diff
@@ -182,10 +231,12 @@ def parse_args() -> argparse.Namespace:
             benchmark_yaml.endswith('yml')), (
                 "--benchmark-yaml needs to take an YAML file.")
 
-  assert bool(benchmark_yaml) != bool(args.benchmarks_directory), (
-      'One and only one of --benchmark-yaml and --benchmarks-directory is '
-      'required. The former takes one benchmark YAML file, the latter '
-      'takes: a directory of them.')
+  assert bool(benchmark_yaml) != bool(
+      args.benchmarks_directory) or args.generate_benchmarks, (
```

Comment:
nit: The logic is not accurate, because it allows taking `--benchmark-yaml`, `--benchmarks-directory`, and `--generate-benchmarks` at the same time.
We need exactly one of them.

Reply:
Fixed this to allow only one